lib/presentation/views/about_us/about_us_info.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class AboutUsInfoView extends StatelessWidget {
+  final String message;
+
+  const AboutUsInfoView({
+    Key? key,
+    this.message =
```

Comment:
Instead of using an optional parameter with default value, this message var could be extracted into a static const inside `AboutUsConsts`. Please refer to `DashboardConsts` for reference

Reply:
need some assistance with this, will pass for now

Reply:
done